index.html

```diff
@@ -13,25 +13,40 @@
         <h1>Welcome to Rock, Paper, Scissors!</h1>
 
         <div class="winner">
-            <h2>Who won</h2>
+            <h2>Choose an option to see who wins</h2>
         </div>
 
         <div class="choices">
-            <div class="button_content">
-                <button><img src="./images/rock.png" alt="Rock"></button>
-                <div class="text">Rock</div>
-                </div>
-            <div class="button_content">
-                    <button><img src="./images/paper.png" alt="Paper"></button>
-                    <div class="text">Paper</div>
+            <div class="content">
+                <button id="Rock" class="selection"><img id="Rock" src="./images/rock.png" alt="Rock">
+                    <div id="Rock" class ="text">Rock</div>
```

Comment:
Id should be unique for each HTML element. Here button and div share an id of "Rock".

Reply:
I agree with your statement about how the ID should be unique for each HTML element. I was running into the issue of the image not being clickable. The text, i.e : "Rock" "Paper" "Scissors", was the only element that was clickable and able to produce the outcome of the h2 querySelector. I managed to fix it by making all the ID's the same. How would I go about fixing it so the image and the name are both selectable but have unique ID's?

Reply:
`//targets button content to give clickable feature
var allButtons = document.querySelectorAll(".content");
allButtons.forEach(buttons => {
    buttons.addEventListener("click", getPlayerChoice); 
});`

So above is the code that attaches the event listener onto all elements with `class="content"`

Then getPlayerChoice(), looks for an id to pass in. You are using the id almost like a parameter. An id should just be a unique name for the element. We should decouple the name and data. 

Try using data attributes. https://www.w3schools.com/tags/att_data-.asp

You can add a data attribute like data-type="rock" onto the div with class="content" then you can pull that data attribute value with something like `this.getAttribute('data-type');` in getPlayerChoice()